.github/workflows/deploy-dev-orange.yml

```diff
@@ -1,17 +1,47 @@
 name: Deploy to Dev-Orange
-run-name: ${{ github.event_name == 'workflow_dispatch' && format('Deploy {0} {1} to Dev-Orange', github.ref_type, github.ref_name) || '' }}
+run-name: ${{ github.event_name != 'pull_request' && format('Deploy {0} {1} to Dev-Orange', github.ref_type, github.ref_name) || '' }}
 
 on:
   workflow_dispatch:
+    inputs:
+      ack:
+        description: "Manual deploys to dev-orange will be wiped out weekly; consider using a PR instead"
+        required: true
+        type: choice
+        options:
+          - I am just deploying main anyway
+          - I am for some other reason okay with that
```

Comment:
I'm hoping that `required: true` with no `default:` will force an explicit selection of one or the other of these before trying to run the workflow. If it doesn’t, I think it'd be more confusing to fail the workflow if this wasn’t acknowledged, so I’ll probably leave a message here in `inputs` with no actual meaning.

Reply:
It does not; with no `default:` a `choice` will default to the first option.